app/components/subcase/subcase-description-panel/subcase-description-edit.js

```diff
@@ -0,0 +1,129 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import CONSTANTS from 'frontend-kaleidos/config/constants';
+import { tracked } from '@glimmer/tracking';
+import { task } from 'ember-concurrency-decorators';
+import { action } from '@ember/object';
+import { saveChanges } from 'frontend-kaleidos/utils/agendaitem-utils';
+
+export default class SubcaseDescriptionEdit extends Component {
+  /**
+   * @argument subcase
+   * @argument onCancel
+   * @argument onSave
+   */
+  @service store;
+  @service newsletterService;
+
+  @tracked subcaseName;
+  @tracked caseTypes;
+  @tracked subcaseType;
+  @tracked caseType;
+  @tracked showAsRemark;
+
+  @tracked isSaving = false;
+
+  constructor() {
+    super(...arguments);
+    this.showAsRemark = this.args.subcase.showAsRemark;
+    this.subcaseName = this.args.subcase.subcaseName;
+    this.loadSubcaseType.perform();
+    this.loadCaseTypes.perform();
+    this.loadCaseType.perform();
+  }
+
+  @task
+  *loadCaseType() {
```

Comment:
Met het oog op discussies omtrent toekomstige refactor van vertrouwelijkheid etc, is het nuttig voor iedereen die hier aan heeft gewerkt om deze https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/70b6a15e89805400917f8fbdacaa421e13bc7b6c/app/config/constants.js#L36 comment te lezen, om goed te beseffen hoe verwarrend deze benaming is, zodat we vermijden dat die in de toekomst verdwijnt, en niet verder gebruikt wordt.

Reply:
De huidige constructie is ook niet echt uitbreidbaar, als er ooit een type zou bijkomen zitten we in de problemen. (er zijn 3 verborgen legacy types).
Die benaming refactoren valt buiten spec, we kunnen daar een ticket voor maken en dan kunnen we de kas nummer al vermelden in een comment bij deze.

Want het zou proper zijn dat we dan ook ineens de URI aanpassen zodat er niet meer `/dossier-type-codes/` instaat.